sass/base/_variables.scss

```diff
@@ -16,6 +16,16 @@ $secondaryColorDisabled: #000000;
 $grey: #5A6978;
 $lightGrey: #B6B6B6;
 
+//gradient colors
```

Comment:
From here to line 24, what are this things doing here? We're not using any blue gradient, as far as I know. Also, we already have background colors (both in normal and light version) some lines down (30 and 31). 

Also are `maintag` and `mainQuestion` variable names accurate? Aren't just other way to use `textDark`?

Reply:
I've just fixed your requested changes =)